migrate_nidirect_file/config/install/migrate_plus.migration.upgrade_d7_file.yml

```diff
@@ -1,4 +1,4 @@
-uuid: 36b1f2a8-ae69-4cdb-917e-34680a84d9f8
+quuid: 36b1f2a8-ae69-4cdb-917e-34680a84d9f8
```

Comment:
Not sure if a typo? The following value looks the same so I'd say it's likely :)

Reply:
It is indeed, good catch